components/dashboard/user-auth-form.tsx

```diff
@@ -1,20 +1,20 @@
-"use client"
+"use client";
 
-import * as React from "react"
-import { useSearchParams } from "next/navigation"
-import { signIn } from "next-auth/react"
-import * as z from "zod"
-import { useForm } from "react-hook-form"
-import { zodResolver } from "@hookform/resolvers/zod"
+import * as React from "react";
```

Comment:
It looks like a few unrelated code style fixes got in. Can you remove these before we commit please? Thank you

Reply:
okay sure🙂